demo/index.html

```diff
@@ -454,23 +454,44 @@
     }());
   </script>
 
-  <p>Here's an image of the line chart:</p>
-
-  <div id='line-chart-img-container'></div>
-
-  <script>
-    (function() {
-      var chart = document.getElementById('line-chart');
-      var imgContainer = document.getElementById('line-chart-img-container');
-
-      chart.addEventListener('google-chart-render', function() {
-        var img = document.createElement('img');
-        img.src = chart.getImageURI();
-        imgContainer.innerHTML = '';
-        imgContainer.appendChild(img);
+  <p>This demonstrates the ways to get the chart image URI:</p>
+
+  <dom-module id="my-el">
+    <template>
+      <p>Original chart:</p>
+      <google-chart
+          id="makeAnImgFromMe"
+          image-uri="{{chartImageUri}}"
+          data='[ ["Label", "Value"],["Memory", 80],["CPU", 55],["Network", 68] ]'></google-chart>
+      <p>Image created via upward data binding:</p>
+      <img id="imgByUpwardBinding" src$="{{chartImageUri}}">
+      <p>Image created via method return value:</p>
+      <img id="imgByMethod">
+    </template>
+    <script>
+      Polymer({
+        is: 'my-el',
+        behaviors: [],
```

Comment:
I'm confused. Why remove this?
I thought the point was to address upward data binding.
That's not possible without a parent element, right?